test/crawler/keyword/keywords_test.exs

```diff
@@ -19,9 +20,9 @@ defmodule Crawler.Keyword.KeywordsTest do
   describe "create_keyword/1" do
     test "given a valid keyword data, creates a keyword" do
       %{id: user_id} = insert(:user)
-      keyword_params = %{name: "keyword", user_id: user_id}
+      keyword = %{name: "keyword", user_id: user_id}
```

Comment:
why to rename it? It is not keyword obj, it is map which is keyword params/attrs

Reply:
Oh, my bad, just forgot to discard this change. Fixed in [d89b6cd](https://github.com/vnntsu/elixir-ic/pull/39/commits/d89b6cd1faf4479251fd071d3edb2ac663c5df1b)